app/mailers/application_mailer.rb

```diff
@@ -11,4 +12,19 @@ def default_url_options(options = {})
       .merge(locale: I18n.locale)
       .merge(options)
   end
+
+  # Add the image file as an attachment
+  # Typically called by a mailer subclass directly
+  def add_email_signature_logo
+    signature_file = File.read("#{Rails.root}/app/assets/images/#{Otis.config.image.hathitrust.email_signature_logo.name}")
```

Comment:
this.hurts.to.read.but.thats.rails.i.guess

Reply:
I was tempted to try `asset_path` but then there's module imports to deal with.

If this kind of thing comes up again we should probably spin off a helper.